spec/lib/pdf_filler/nj1040_pdf_spec.rb

```diff
@@ -89,15 +89,15 @@
         end
 
         it 'leaves married filing separately spouse SSN fields blank' do
-          expect(pdf_fields["undefined_7"]).to eq ""
-          expect(pdf_fields["undefined_8"]).to eq ""
-          expect(pdf_fields["Enter spousesCU partners SSN"]).to eq ""
-          expect(pdf_fields["Text31"]).to eq ""
-          expect(pdf_fields["Text32"]).to eq ""
-          expect(pdf_fields["Text33"]).to eq ""
-          expect(pdf_fields["Text34"]).to eq ""
-          expect(pdf_fields["Text35"]).to eq ""
-          expect(pdf_fields["Text36"]).to eq ""
+          expect(pdf_fields["undefined_7"]).to eq nil
```

Comment:
Trying to wrap my head around why this changed. So these are all nil by default in the new PDF, whereas before the PDF had an empty string value in the field?

Reply:
the best clue I have is the output from `pdftk`. compare the first two fields

from 2023:
```
---
FieldType: Text
FieldName: undefined
FieldFlags: 12582912
FieldValue:
FieldJustification: Center
FieldMaxLength: 1
---
```

and from 2024:
```
---
FieldType: Text
FieldName: undefined
FieldFlags: 12582912
FieldValue: 
FieldJustification: Center
FieldMaxLength: 1
---
```

identical **except** there is an extra space in the `FieldValue` of the 2024 version. I think this is related to the difference in code too, though I'm not sure why.